server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java

```diff
@@ -46,7 +47,7 @@
   }
 
   void deleteAll() {
-    session.wsClient().users().search(SearchRequest.builder().build()).getUsersList()
+    session.wsClient().usersOld().search(SearchRequest.builder().build()).getUsersList()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "usersOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)